modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc

```diff
@@ -1,15 +1,84 @@
-= Troubleshooting the Runtime Manager Agent
+= Troubleshoot the Runtime Manager Agent
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: agent, mule, servers, monitor, notifications, external systems, third party, get status, metrics
+:keywords: agent, mule, servers, monitor, notifications, external systems, get status, metrics
 
 image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
 image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot the Anypoint Runtime Manager agent installation issues. If you experience issues with the Runtime Manager agent plug-in or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.
+Connection issues between your on-premises Mule runtime engine and the Runtime Manager agent might cause discrepancies, such as metrics not appearing in the server monitoring dashboard in Runtime Manager.
+
+== Troubleshoot Connection Issues Between the Agent and Mule
+
+To troubleshoot connection issues:
+
+. Check the JVM you are using.
++
+If you are using the IBM JVM, you must install the truststore manually because this truststore requires different parameters than Oracle JVM or OpenJDK.
```

Comment:
Will "truststore" be familiar to a newbie user? Or does this document assume that a user will be more advanced?

Reply:
The link gives more info on the truststore.